sematic/ui/packages/common/src/pages/RunDetails/RunSection.tsx

```diff
@@ -0,0 +1,44 @@
+import styled from '@emotion/styled';
+import Box from '@mui/material/Box';
+import { useTheme } from '@mui/material/styles';
+import Typography, { typographyClasses } from '@mui/material/Typography';
+import MoreVertButton from 'src/component/MoreVertButton';
+import RunsDropdown from 'src/component/RunsDropdown';
+import theme from 'src/theme/new';
+import TagsList from 'src/component/TagsList';
+
+const Section = styled.section`
+    margin-bottom: ${theme.spacing(3)};
+`;
+
+const StyledVertButton = styled(MoreVertButton)`
+    transform: translate(50%,0);
+`;
+
+const BoxContainer = styled(Box)`
+    display: flex;
+    & .${typographyClasses.root} {
+        margin-right: ${theme.spacing(2)};
+    }
+`
+const RunSection = () => {
+    const theme = useTheme();
+
+    return <Section>
+            <h2>RUN</h2>
+            <BoxContainer style={{marginBottom: theme.spacing(3)}}>
+                <RunsDropdown />
+                <StyledVertButton />
+            </BoxContainer>
+            <BoxContainer style={{marginBottom: theme.spacing(2)}}>
+                <Typography variant='small'>Developer E.</Typography>
+                <Typography variant='small'>CloudResolver</Typography>
+            </BoxContainer>
+            <BoxContainer>
+                <TagsList tags={['example', 'torch', 'mnist']} />
```

Comment:
Are the tags supposed to be clickable?
You could link to the run filter page, with the tag filter already populated with the clicked tag.

Reply:
Yeah, they will be clickable in the future, currently, since we don't have the run filter page yet, it is a bit hard for me to make up links. The current state of the PR first concentrates on the visual styles, the business logic/interactivity will be considered in the follow-up PRs when they are easier to implement. Thanks for mentioning it though!